### ISSUE TITLE
Weird timeline reset when scrolling through the timeline too fast

### ISSUE THREAD

**Issue by tsuza** [paulpacifico/shutter-encoder#204]
If you scroll through the timeline slightly too fast, the timeline will reset to the previous position ( most of the time, otherwise it'll reset to the last ""recorded"" position while you were scrolling ). It'll work fine if you do it slow though.

Here below I've provided a GIF to visually show what I mean:
![javaw_7zsalzEn5S](https://github.com/paulpacifico/shutter-encoder/assets/73082112/43384706-8556-41e9-a5b7-ed2f5c54497f)

I assume that the application is setting the timeline's position variable way too slowly, which creates this inconsistency compared to what I'm actually seeing.

> **O.S.**: Windows 11.
> **App Version**: 18.2

**Comment by paulpacifico** [comment]
Yes because it stops on what your seeing, meaning that the input/output points are valid when the frame is loaded.

**Comment by tsuza** [comment]
I don't think that's a desired outcome, though. That's not how it works in any other video editing software ( e.g. Davinci ). It just creates annoyances when you're trying to trim a video.

**Comment by tsuza** [comment]
To add to this, if you try to change the duration from the right ( I genuinely don't know how it's called... the right bracket for trimming, I hope you get it ), the video completely hangs and you can't interact with it anymore ( neither watching it nor trimming it ). Manually editing the values works fine though.
